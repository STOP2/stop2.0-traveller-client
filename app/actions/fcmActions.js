export const SET_FCM_TOKEN = 'SET_FCM_TOKEN'

export let setFCMToken = function(token)
{
    return {
        type: SET_FCM_TOKEN,
        token: token
    }
}
