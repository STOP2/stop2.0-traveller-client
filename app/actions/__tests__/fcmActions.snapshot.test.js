import { setFCMToken } from '../fcmActions';

describe('fcm actions', () => {
  it('creates a SET_FCM_TOKEN action', () => {
    expect(setFCMToken(12345)).toMatchSnapshot();
  });
});
