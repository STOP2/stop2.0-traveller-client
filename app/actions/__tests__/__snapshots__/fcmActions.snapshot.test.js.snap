exports[`fcm actions creates a SET_FCM_TOKEN action 1`] = `
Object {
  "token": 12345,
  "type": "SET_FCM_TOKEN",
}
`;
